include/roboteam_world/WorldHandler.h

```diff
@@ -16,8 +16,8 @@ class WorldHandler {
   proto::Publisher<proto::SSL_Referee> *ref_pub;
   proto::Publisher<proto::SSL_GeometryData> *geom_pub;
 
-  WorldBase *world;
-  KalmanFilter *KF;
+  double lastPacketTime;
```

Comment:
What is the format of time? Millis? Seconds? at least clarify this in a comment, and think if a double is the right type for this.

Reply:
Double is what we were using before. We should probably use long ints as nanoseconds. Timing is a massive problem in our system in general but I don't think I should should solve that in this PR. The way I do it now is ok-ish. The problem is in previous world branches we were throwing away timing information. (Also: "everything is in SI units unless otherwise specified"  is one of our standards , right? Though I agree clarifying comments cannot hurt in general.)